examples/streamed-list-objects/src/main/java/dev/openfga/sdk/example/StreamedListObjectsExample.java

```diff
@@ -0,0 +1,194 @@
+package dev.openfga.sdk.example;
```

Comment:
Reference for this : https://github.com/openfga/dotnet-sdk/pull/156/files#diff-22fba32053e04d6f19bf16b6e2cd57cfc95dcef30765760c761112acad075ad4